enterprise/api/v1/serializers.py

```diff
@@ -590,6 +590,8 @@ class EnterpriseGroupSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.EnterpriseGroup
         fields = ('enterprise_customer', 'name', 'uuid')
+    
+    enterprise_customer = EnterpriseCustomerSerializer()
```

Comment:
I think we should probably have a test to ensure this is giving us the values we want